integration/spark2/src/main/scala/org/apache/carbondata/spark/util/CarbonSparkUtil.scala

```diff
@@ -117,4 +116,18 @@ object CarbonSparkUtil {
     case _ =>
       delimiter
   }
+  def getKeyOnPrefix(path: String): (String, String, String) = {
+    val endPoint = "spark.hadoop." + ENDPOINT
+    if (path.startsWith(CarbonCommonConstants.S3A_PREFIX)) {
+      ("spark.hadoop." + ACCESS_KEY, "spark.hadoop." + SECRET_KEY, endPoint)
```

Comment:
Duplicated  spark.hadoop."  literals make the process of refactoring error-prone, since you must be sure to update all occurrences."，I think you can  define a variable uniformly.